astrophot/image/image_header.py

```diff
@@ -463,42 +301,58 @@ def get_state(self):
         return state
 
     def _save_image_list(self):
+        """
+        Constructs a fits header object which has the necessary information to recreate the Image_Header object.
+        """
         img_header = fits.Header()
         img_header["IMAGE"] = "PRIMARY"
-        img_header["PXLSCALE"] = str(self.pixelscale.detach().cpu().tolist())
         img_header["WINDOW"] = str(self.window.get_state())
         if not self.zeropoint is None:
             img_header["ZEROPNT"] = str(self.zeropoint.detach().cpu().item())
         if not self.note is None:
             img_header["NOTE"] = str(self.note)
         return img_header
 
+    def set_fits_state(self, state):
+        """
+        Updates the state of the Image_Header using information saved in a fits header.
```

Comment:
Yes, it can be just a dict. This function in particular uses FITS keywords that I've defined myself so that I can keep track of all the data. At some point I should look up how to read this information from a standard FITS format header.